### ISSUE TITLE
Disabled TextView text color inconsistent with Material Design specs

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#1567]
### Problem description
According to the [Material Design guidelines](https://material.io/guidelines/style/color.html#color-usability) (see _Text on backgrounds_ section), disabled text should have 38% opacity on light backgrounds and 50% opacity on dark backgrounds. Disabled text color does not adhere to these specs.

In the following snippet:
  * disabled text view with white text color appears completely black on the black background, which makes it invisible, although the specs recommend 50% opacity of the text
  * disabled text view with black text color appears with a lighter color than the specified 38% opacity

### Expected behavior

Maybe we should let the user specify the disabled text color himself:
  * by not modifying the text color at all when `enabled` is set to `false` (then the user can just set `textColor` accordingly). This is how e.g. the background color of Composite is handled.
  * by introducing a new property `disabledTextColor`

### Environment

- Tabris.js version: 2.2.0
- Device: Emulator
- OS: Android 8.0

### Code snippet

```javascript
const {Composite, TextView, ui} = require('tabris');

ui.contentView.append(
  new Composite({
    left: 0, top: 'prev()', right: 0,
    background: 'black'
  }).append(
    new TextView({
      left: 0, top: 'prev()', right: 0,
      font: 'black 18px',
      text: 'Disabled', textColor: 'white', enabled: false}),
    new TextView({
      left: 0, top: 'prev()', right: 0,
      font: 'black 18px',
      text: '38% opacity', textColor: 'rgba(255, 255, 255, 0.38)'
    }),
    new TextView({
      left: 0, top: 'prev()', right: 0,
      font: 'black 18px',
      text: '50% opacity', textColor: 'rgba(255, 255, 255, 0.5)'
    })
  ),
  new Composite({
    left: 0, top: 'prev()', right: 0,
    background: 'white'
  }).append(
    new TextView({
      left: 0, top: 'prev()', right: 0,
      font: 'black 18px',
      text: 'Disabled', textColor: 'black', enabled: false}),
    new TextView({
      left: 0, top: 'prev()', right: 0,
      font: 'black 18px',
      text: '38% opacity', textColor: 'rgba(0, 0, 0, 0.38)'
    }),
    new TextView({
      left: 0, top: 'prev()', right: 0,
      font: 'black 18px',
      text: '50% opacity', textColor: 'rgba(0, 0, 0, 0.5)'
    })
  )
);
```

**Comment by mpost** [comment]
The point is valid. We will update the Android client to use the user provided color with alpha applied as the disabled color.

**Comment by mpost** [comment]
Fixed as described above for the 2.3 release.
